client/src/App.js

```diff
@@ -1,7 +1,17 @@
 import React from "react";
+import { Routes, Route } from "react-router-dom";
+import LoginPage from "./pages/LoginPage/LoginPage.js";
+import SignUpPage from "./pages/SignUpPage/SignUpPage.js";
 
 const App = () => {
-  return <div>UTimeManager</div>;
+  return (
+    <div>
+      <Routes>
```

Comment:
The routes look good although I would consider also putting in a redirect route such that any invalid routes would bring the user back to the main page

Reply:
The following link would be good to take a look at, once the authentication is fully implemented.
https://reactrouter.com/docs/en/v6/examples/auth

Reply:
> 

Should we just redirect to a page not found route? I don't see a natural "main page" in our application.

Reply:
Yep that also works as well!

A natural main page could be for example, the login page (if you're currently logged out) or if you're signed in, then it can directly bring you back to the main calendar portion where you see all your current tasks for the week

Reply:
Fixed: Implemented PageNotFound as the no-match route. If there is time in the future, we will implement conditional routing to login page (if not logged in) or calendar page (if logged in) if a page doesn't exist.